frontend/src/pages/videos/VideoCard.tsx

```diff
@@ -18,12 +21,52 @@ const useStyles = makeStyles(() => ({
 function VideoCardPage() {
```

Comment:
These two function will be responsible of retrieving the criteria and the related scores. Having such functions outside of `VideoCardPage` page will ease the future development (see my other comments below).

They are not perfect, as they still contain the type `any` which should be replaced by a proper type definition if possible. 

```suggestion

const getVideoCriteria = (criteriaScores: Array<any>) => {
  return Object.fromEntries(
    criteriaScores.map((criteria: any) => [
      criteria.criteria,
      criteria.criteria,
    ])
  );
};

const getVideoMetaData = (criteriaScores: Array<any>) => {
  return {
    data: Object.fromEntries(
      criteriaScores.map((criteria: { criteria: string; score: number }) => [
        criteria.criteria,
        criteria.score,
      ])
    ),
    meta: { color: 'purple' },
  };
};

function VideoCardPage() {
```